sdks/python/apache_beam/io/iobase.py

```diff
@@ -910,6 +913,67 @@ def from_runner_api_parameter(parameter, context):
     Read.from_runner_api_parameter)
 
 
+class SDFBoundedSourceWrapper(ptransform.PTransform):
+  """A ``PTransform`` that uses SDF to read from a ``BoundedSource``.
+  NOTE: This transform can only be used with beam_fn_api enabled.
```

Comment:
docstring format should be 
```
one liner.

additional details.
```

The blank line after the first line is important.